### PR TITLE
bitfinex: fix websocket subscribing issue caused by formatting

### PR DESCRIPTION
**Pull request by shazbert**
# PR Description

Changes formatting so subscribe works for websocket.

Fixes # (issue)

## Type of change

Please delete options that are not relevant and add an `x` in `[]` as item is complete.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How has this been tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration and
also consider improving test coverage whilst working on a certain feature or package.

- [ ] go test ./... -race
- [ ] golangci-lint run
- [ ] Test X

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation and regenerated documentation via the documentation tool
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally and on Github Actions/AppVeyor with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by gloriousCode — CHANGES_REQUESTED** [review]
It works!

File: exchanges/bitfinex/bitfinex_websocket.go [hunk 1]
```diff
@@ -1526,7 +1526,7 @@ func (b *Bitfinex) GenerateDefaultSubscriptions() ([]stream.ChannelSubscription,
 					}
 					params["key"] = "trade:1m:" + prefix + enabledPairs[k].String() + fundingPeriod
 				} else {
-					params["symbol"] = enabledPairs[k].String()
+					params["symbol"] = enabledPairs[k].Format(currency.EMPTYFORMAT).Upper().String()
```
[the remark this answers is not in the document]
Comment (shazbert) [reply exchanges/bitfinex/bitfinex_websocket.go]: Had no impact on allocation or speed, but it was a visual improvement, scoped it locally.

**Review by thrasher- — APPROVED** [review]
tACK

**Review by gloriousCode — APPROVED** [review]
tAcK1!

**Pull request merged**